keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -112,7 +109,7 @@ apiVersion: kubelet.config.k8s.io/v1beta1
 featureGates:
   ...
   CPUManagerPolicyOptions: true
-  CPUManagerPolicyAlphaOptions: true
+  CPUManagerPolicyBetaOptions: true
```

Comment:
this is the intended behavior so no need to call it out. It is true unless user explicitely disable it.

Reply:
ok, got you.